source/Ship.h

```diff
@@ -250,12 +250,19 @@ class Ship : public Body, public std::enable_shared_from_this<Ship> {
 	bool IsFleeing() const;
 	// Check if this ship is currently able to begin landing on its target.
 	bool CanLand() const;
+	// What kind of action this is we are trying to do.
+	enum ActionType {AFTERBURNER, BOARD, COMMUNICATION, FIRE, PICKUP, SCAN};
```

Comment:
enum class?

Reply:
idk look at line 90 of Mission.h it's just enum

Reply:
```suggestion
	enum class ActionType {AFTERBURNER, BOARD, COMMUNICATION, FIRE, PICKUP, SCAN};
```